support/modules-artifact-gen/docker-artifact-gen

```diff
@@ -123,6 +132,11 @@ if [ -z "${IMAGES}" ]; then
   show_help_and_exit_error
 fi
 
+# Create required files for the Update Module
+tmpdir=$(mktemp -d)
+run_args_file="$tmpdir/run_args"
+echo ${run_args} > ${run_args_file}
```

Comment:
Since this is just a small piece of information, I think it's better if this is added to the meta data under the `run_args` key. It omits the need for an extra file, and also has the advantage of being visible in the UI.